mock-certify-plugin/src/main/java/io/mosip/certify/util/PKCS12Reader.java

```diff
@@ -0,0 +1,49 @@
+package io.mosip.certify.util;
+
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.ByteArrayInputStream;
+import java.security.*;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+import java.security.cert.X509Certificate;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.util.Base64;
+
+@Slf4j
+public class PKCS12Reader {
+    public KeyPairAndCertificate extract(String keyCert) {
```

Comment:
is the keyCert really in pkcs12 format ?

Reply:
No, its not extracting from p12 , its extracting key and certificate from encoded format. Will rename as KeyPairAndCertificateExtractor